agents/unix/meson.build

```diff
@@ -463,7 +472,7 @@ executable('ta', sources, generated_sources,
            include_directories: includes,
            c_args: c_args,
            link_args: link_args,
-           dependencies: deps)
+           dependencies: deps + ta_external_deps)
```

Comment:
In fact it is a wrong direction.
The key problem here is that when ta_rcf_listener is introduced it breaks TA abstraction from underlying communication protocol. In theory, it could be anything, but rcf_comm_agent_create_listener() hardwires it to TCP over IPv4.

The correct fix is to build ta_rcf_listener if and only if comm_net_agent is present in TA libraries  )agent-unix-libs) and do not add anything except required minimum (logger libs) plus comm_net_agent to ta_rcf_listener dependencies. It does not need any RPC and CS libraries as I understand.